server/routerlicious/Dockerfile

```diff
@@ -26,8 +27,8 @@ WORKDIR /usr/src/server
 
 # Copy over the package and package-lock and install prior to the other code to optimize Docker's file system cache on rebuilds
 COPY package*.json ./
-COPY lerna.json .
-COPY lerna-package-lock.json .
+COPY pnpm*.yaml ./
+COPY scripts/*.* ./scripts/
```

Comment:
revert this and the lines below. Also, why is this change seaprate from the other PR? I would change it all at the same time.

Reply:
@CraigMacomber suggested doing it in a separate PR

Reply:
I don't see a reason to update docker's base image at the same time as Node, especially when node is having issues and we arn't sure if that updates is related to the docker image or not.

That said, these are all server code. Can we just delate all this instead of updating it since none of server is used on this branch?

Reply:
Or at least disable those pipelines